homework-submission/myapp/src/components/header.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import logo from "../logo.svg";
+
+class Header extends React.Component {
+  render() {
+    return (
+      <div className={this.props.className}>
```

Comment:
you hard coded it in `App.js` you can put it here simply 😄 instead of passing it as `props`